### DISCUSSION TITLE
Sports replays

### DISCUSSION THREAD

**Discussion by sjvis** [ExistentialAudio/BlackHole#813] [Q&A]
I tend to watch recorded replays of sporting events/matches from a library on a website. On some of the replays I watch, I can screen record with audio using Quicktime and Blackhole with no issue. On other replays, I can screen record but the audio is missing (It's there but barely ... very, very low and unintelligible). Could the way the event is broadcast and saved on the website impact how Blackhole can capture the audio when replayed, even though the replay looks and sounds fine when I watch it on my computer? Any suggestions? Thanks.

**Comment by devinroth** [comment]
Hard to answer that question. There have been some apps that have intentionally blocked BlackHole. No idea if that’s the case here.

Another issue I’ve seen is apps controlling the volume of audio devices. That could affect BlackHole.

And lastly, if you’re using an aggregate device, some applications handle that differently. I’ve seen situations where the volume has been reduced because there were more than 2 channels on a device and the application chose to do some odd stuff.
